packages/matter.js/src/cluster/server/AdministratorCommissioningServer.ts

```diff
@@ -198,11 +198,13 @@ class AdministratorCommissioningManager {
      * This method is used internally when the commissioning window timer expires or the commissioning was completed.
      */
     private endCommissioning() {
-        logger.debug("End commissioning window.");
         if (this.commissioningWindowTimeout !== undefined) {
             this.commissioningWindowTimeout.stop();
             this.commissioningWindowTimeout = undefined;
+        } else {
```

Comment:
Trivial - should invert the tests and just return if this.commissioningWindowTimeout === undefined